src/pages/preferences/preferencesTabs/notificationTab/NotificationField.tsx

```diff
@@ -57,22 +57,23 @@ const GridContainer = styled.div`
 const NotificationMethod = styled.div`
   display: flex;
   align-items: center;
-  gap: ${size.l};
+  margin-bottom: ${size.xs};
```

Comment:
Just for context were there design changes that made this necessary or was this just an improvement we would have done anyways?

Reply:
they added a flex-box around the radio-group which messed up the styles for all of our radio groups so they needed to be updated!